### PR TITLE
Get distance factor based on machine units and interp units

### PR DESCRIPTION
**Pull request by dkhughes**
Address #122

### CODE REVIEW

File: src/application/ApplicationHelper.qml [hunk 1]
```diff
@@ -27,8 +27,20 @@ Item {
     property var axisNames: getAxisNames()
     property string timeUnits: getTimeUnits()
     property string distanceUnits: getDistanceUnits()
+    property string machineUnits: getMachineUnits()
     property double timeFactor: (timeUnits === "min") ? 60 : 1
-    property double distanceFactor: (distanceUnits === "mm") ? 1.0 : ((distanceUnits === "in") ? 0.03937 : 0.1)
+    property double distanceFactor: {
+            if(machineUnits == "mm") {
```
Comment (machinekoder) [inline src/application/ApplicationHelper.qml]: use `===` in JavaScript

**Comment by machinekoder** [comment]
The CI for Android fails because the PR is from a different repository. I have to fix this issue but nothing to worry for this PR.

File: src/application/ApplicationHelper.qml (hunk 1 above)
Comment (dkhughes) [inline src/application/ApplicationHelper.qml]: Good call, fixed. This was one of the earlier files I wrote in Javascript.

**Comment by machinekoder** [comment]
@dkhughes Thank you, excellent work!

**Pull request merged**